server/data/educationAndWorkPlanApiMapper.ts

```diff
@@ -0,0 +1,22 @@
+import type { CreateGoalRequest, CreateStepRequest } from 'educationAndWorkPlanApiClient'
+import type { CreateGoalDto } from 'dto'
+
+export default class EducationAndWorkPlanApiMapper {
```

Comment:
I guess we need to think about where our mappers should live 🤔 
I guess this is OK in `data` because its creating an object that will be used by a client class (that also lives in `data` - yuck! 🤣 )
But I wonder if perhaps we should put it in `data/mappers` or similar ?